selenium/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "axe-core-maven-html-selenium",
-  "version": "4.8.2",
+  "version": "4.9.0",
```

Comment:
Separate issue - odd it's bumping the lock file here - it's not a particularly a big problem but we should track an issue to figure out why the package json is tagged with X version but the lock file isn't being updated to match the new version.

Reply:
It looks like lock files were versioned but incorrectly I can revert this and do this in another PR to properly version playwright/selenium so that in the future semantic version should be doing it for us automagically